### PR TITLE
OpenAPI generator tutorial

### PR DESCRIPTION
**Pull request by nicoongit**

### CODE REVIEW

File: docs/sdks/specification.md [hunk 1]
```diff
@@ -1,2 +1,157 @@
...
+
+There are several ways to customise an SDK created by the OpenAPI Generator:
+
+* Using a configuration file
+* Using command line arguments
+* Using mustache templates
+
+##### Configuration file
+
+There is anumber of configuration options that the OpenAPI Generator supports for [Ruby](https://openapi-generator.tech/docs/generators/ruby#config-options).
```
Comment (tsolakoua) [inline docs/sdks/specification.md:76]: anumber -> a number

File: docs/sdks/specification.md [hunk 2]
```diff
@@ -1,2 +1,157 @@
...
+  -i /local/CitySearch_v1_swagger_specification.json \
+  -g ruby --model-name-prefix Amadeus \
+  -o /local/city_search_ruby
+```
+
+In this way, all models in our SDK will be prefixed with `Amadeus`, e.g. `AmadeusComponent`.
+
+### Conclusion
+
+In this tutorial we have seen how easy it is to generate a basic SDK from our API specification files using the OpenAPI Generator. We took Ruby as an example, but the OpenAPI Generator supports [many other languages](https://openapi-generator.tech/docs/generators), so you can easily find a solution that meets yor requirements.
```
Comment (tsolakoua) [inline docs/sdks/specification.md:157]: yor -> your

**Review by tsolakoua — APPROVED** [review]
Apart from the very small typo, it looks good to me <3

**Comment by nicoongit** [comment]
> Apart from the very small typo, it looks good to me <3

Thanks Anna, corrected the typo.

**Pull request merged**